tests/test.sh

```diff
@@ -2,18 +2,33 @@
 
 set -e # Fail if any line fails
 
-# 1. Run unit tests
+printf "1. Run unit tests\n"
+printf "==============================================\n"
 python -m unittest tests/e3sm_diags/*/test_*.py
 
-# 2. Enter tests directory
-cd tests
-
-# 3. Copy over test data and images from the web. [Takes about four minutes]
-cd integration && python download_data.py && cd ..
+printf "\n2. Copy over test data and images from the web\n"
+printf "==============================================\n"
+# Takes about four minutes
+python -m tests.integration.download_data
 
 # To use `scp`, run the following two lines instead. Be sure to change <username>. [Takes about two minutes]
 #scp -r <username>@blues.lcrc.anl.gov:/lcrc/group/e3sm/public_html/e3sm_diags_test_data/integration/integration_test_data integration_test_data
 #scp -r <username>@blues.lcrc.anl.gov:/lcrc/group/e3sm/public_html/e3sm_diags_test_data/integration/expected/integration_test_images integration_test_images
 
-# 4. Run integration tests
-cd integration && python -m unittest && cd ..
```

Comment:
The reason why this worked before was because there was no command line argument being specified for running tests using a specific regex pattern (`test_*.py`). It was just `cd`ing into the `tests/integration` directory and running all tests in there.